rc/lsp.kak

```diff
@@ -1415,6 +1423,49 @@
 " | eval "${kak_opt_lsp_cmd} --request") > /dev/null 2>&1 < /dev/null & }
 }
 
+define-command -hidden lsp-indent-guides-request -docstring "lsp-indent-guides-request: request updating lsp_indent_guides for the buffer" %{
+  declare-option -hidden int lsp_indent_guides_timestamp -1
+  nop %sh{
+    if [ $kak_opt_lsp_indent_guides_timestamp -eq $kak_timestamp ]
+    then
+      exit
+    fi
+
+    indent_guides_characters="[ $(
+      DELIM=""
+      for CHARACTER in $(echo $kak_quoted_opt_lsp_indent_guides_characters)
+      do
+        printf "%s %s" "$DELIM" "$CHARACTER"
+        DELIM=","
+      done
+    ) ]"
+
+    (printf %s "
+session  = \"${kak_session}\"
+buffile  = \"${kak_buffile}\"
+filetype = \"${kak_opt_filetype}\"
+version  = ${kak_timestamp:-0}
+method   = \"kak-lsp/indent-guides\"
+$([ -z ${kak_hook_param+x} ] || echo hook = true)
+[params]
+skip = $kak_opt_lsp_indent_guides_skip
+highlight_focused = $kak_opt_lsp_indent_guides_highlight_focused
+characters = $indent_guides_characters
+position_line = $kak_cursor_line
+" | eval "${kak_opt_lsp_cmd} --request") > /dev/null 2>&1 < /dev/null & }
+}
+
+define-command lsp-indent-guides-enable -params 1 -docstring "lsp-indent-guides-enable <scope>: enable indent guides for <scope>" %{
```

Comment:
I wonder if we should make it the default.
Either way, maybe add a line in the Usage section of our README.

Reply:
Yeah, agree that it should be reflected in the README. Will do